ontotext-yasgui-web-component/src/services/utils/html-elements-util.ts

```diff
@@ -37,4 +37,22 @@ export class HtmlElementsUtil {
   static getControlBar(hostElement: HTMLElement): HTMLElement {
     return hostElement.querySelector('.yasgui .controlbar');
   }
+
+  static addHiddenClass(hostElement: HTMLElement, ...elementSelectors: string[]) {
+    elementSelectors.forEach((elementSelector) => {
+      const element = hostElement.querySelector(elementSelector);
+      if (element) {
+        element.classList.add('hidden');
+      }
+    });
+  }
+
+  static removeHiddenClass(hostElement: HTMLElement, ...elementSelectors: string[]) {
```

Comment:
I'd rather implemented a single utility method like these but using the classList.toggle('hidden'). I honestly this that having both methods is too much

Reply:
Done